parameters/clic.yaml

```diff
@@ -44,12 +44,12 @@ setup:
   train: yes
   weights:
   weights_config:
-  lr: 0.0001
-  num_epochs: 500
+  lr: 0.001
+  num_epochs: 100
   dtype: float32
   trainable:
   classification_loss_type: sigmoid_focal_crossentropy
-  lr_schedule: none  # cosinedecay, exponentialdecay, onecycle, none
+  lr_schedule: onecycle  # cosinedecay, exponentialdecay, onecycle, none
```

Comment:
Using the TF model, I get slightly better results using cosinedecay over onecycle.